lib/menu/prayer_groups_page.dart

```diff
@@ -23,6 +25,12 @@ class _PrayerGroupsPageState extends State<PrayerGroupsPage> {
 
   Future<void> _loadData() async {
     try {
+      if(!Platform.isLinux){
```

Comment:
```suggestion
      if (Platform.isAndroid) {
```
Vagy ha ezek a permission-ök nem csak Androidon kellenek, akkor jobb lenne átnevezni a függvényeket. Mindenesetre szerencsésebb pozitív platform check-eket csinálni, a többi hasonló helyen `Platform.isAndroid || Platform.isIOS` van.